### PR TITLE
OCPBUGSM-18082 - Fix install command failed test

### PR DESCRIPTION
**Pull request by slaviered**
/assign @filanov 
/assign @YuviGold

### CODE REVIEW

**Comment by avishayt** [comment]
Is this part of implementing OCPBUGSM-18082 or should it be a separate bug?
Also can you please add a couple of words about what the issue is here?

File: subsystem/cluster_test.go [hunk 1]
```diff
@@ -851,14 +851,18 @@ var _ = Describe("cluster install", func() {
 			Expect(swag.StringValue(c.Status)).Should(Equal(models.ClusterStatusInstalling))
 			Expect(swag.StringValue(c.StatusInfo)).Should(Equal("Installation in progress"))
 			Expect(len(c.Hosts)).Should(Equal(5))
+			var master *models.Host
 			for _, host := range c.Hosts {
 				Expect(swag.StringValue(host.Status)).Should(Equal(models.HostStatusInstalling))
+				if master == nil && host.Role == models.HostRoleMaster {
+					master = host
```
Comment (avishayt) [inline subsystem/cluster_test.go:858]: @YuviGold @filanov Assigning a pointer to the iterator is not safe, right?

Comment (slaviered) [reply subsystem/cluster_test.go:858]: good catch. I will just copy the ID

Comment (filanov) [reply subsystem/cluster_test.go:858]: @avishayt you are right - it will point to the last host.

File: subsystem/cluster_test.go [hunk 2]
```diff
@@ -851,14 +851,18 @@ var _ = Describe("cluster install", func() {
 			Expect(swag.StringValue(c.Status)).Should(Equal(models.ClusterStatusInstalling))
 			Expect(swag.StringValue(c.StatusInfo)).Should(Equal("Installation in progress"))
 			Expect(len(c.Hosts)).Should(Equal(5))
+			var masterID *strfmt.UUID
```
Comment (filanov) [inline subsystem/cluster_test.go:854]:
```suggestion
			var masterID strfmt.UUID
			for _, host := range c.Hosts {
				Expect(swag.StringValue(host.Status)).Should(Equal(models.HostStatusInstalling))
				if masterID.String() == "" && host.Role == models.HostRoleMaster {
					masterID = *host.ID
				}
			}
```

Comment (slaviered) [reply subsystem/cluster_test.go]: DONE

**Review by filanov — APPROVED** [review]

**Comment by filanov** [comment]
> Is this part of implementing [OCPBUGSM-18082](https://issues.redhat.com/browse/OCPBUGSM-18082) or should it be a separate bug?
> Also can you please add a couple of words about what the issue is here?

@avishayt the issue is that we don't fail cluster installation if one of the workers is failed to install, so in this test random host was selected to report installation failure and because @slaviered added another worker the probability of the failure is higher. 

@slaviered usually when fixing something like this you should add a short explanation to the commit message.

File: subsystem/cluster_test.go (hunk 1 above)
Comment (YuviGold) [reply subsystem/cluster_test.go]: Good catch

**Review by avishayt — APPROVED** [review]

**Pull request merged**